odooconnector_base/unit/backend_adapter.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# © 2015 Malte Jacobi (maljac @ github)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+
+import oerplib.error
```

Comment:
yes.
You should probably log the error as well (there is probably some standard code in some OCA module for this kind of import)